notebooks/TestGrid_EDA.ipynb

```diff
@@ -89,6 +89,7 @@
        " '\"redhat-openshift-okd-release-4.6-blocking\"',\n",
```

Comment:
<p>Hi Michael, </p><p> </p><p>Is it possible to map test case name and date on this figure. i.e. on X axis we can have date and on y test case name.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/23/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
<p>yes, it is possible to add some additional labels to this graph. But the intention here is just to show the grid we have matches the image. But the data for time and test name is still available, I was worried it might over-crowed the plot though so I left it out. </p>